docs/changelogs/vNEXT.rst

```diff
@@ -39,3 +39,4 @@ Other bugs & improvements
 - Improved performance of policies with many rules having ground (constant) parameters.
 - Improved performance of ``in`` operator (list membership) with many ground elements.
 - Stack traces return the original policy source instead of the internal version.
+- New ffi methods for passing printing and warning messages from rust to app languages.
```

Comment:
Is the change here that messages that weren't showing up before will now show up?

Reply:
There's no user facing change.

Reply:
I guess I was thinking of the changelog as more user-facing stuff (including under-the-hood details that impact the user experience like perf improvements), but I don't have any objection to including stuff like this.

Reply:
The user-facing part would be that we use the "correct" thing for writing messages - like `console.warn` for TS instead of printing to stderr.

We could maybe use `logger` for python as well, for example.